Subtester/Assets/Scenes/BuyButton.prefab

```diff
@@ -1,22 +1,12 @@
 %YAML 1.1
 %TAG !u! tag:unity3d.com,2011:
---- !u!1001 &100100000
```

Comment:
adjusting sizing, font, positioning, etc. see the video for details on how this looks now